docs/src/languages/cpp.md

```diff
@@ -158,3 +156,15 @@ You can use CodeLLDB or GDB to debug native binaries. (Make sure that your build
   }
 ]
 ```
+
+## Extensions
+
+Zed implements several clangd [extensions](https://clangd.llvm.org/extensions) to enhance the support for C/C++ development.
```

Comment:
```suggestion
Zed implements several `clangd` [extensions](https://clangd.llvm.org/extensions) to enhance the support for C++ development.
```

Reply:
No, the two languages differ greatly, even if there are similarities in certain areas.

Reply:
While I agree the languages differ in many ways, their support in Zed is practically identical:

- Shared tooling: Both languages are powered by `clangd`
- Features Parity: e.g. these extensions works exactly the same for both C and C++.
- The setup/configuration process applies on both languages

Since these docs focus on language server setup and editor features rather than language syntax, merging them could help avoid duplication and keep the instructions consistent for all C-family developers.